src/components/Estamp/Home/components/Profile.tsx

```diff
@@ -0,0 +1,55 @@
+import { HomeIcon, PencilSquareIcon } from '@heroicons/react/24/solid';
+import placeHolder from '@/public/images/pfp-placeholder.svg';
+import Image from 'next/image';
+import { useAuth } from '@/context/AuthContext';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import { getBaan } from '@/utils/baan';
+import { IUser } from '@/types/user';
+const Profile = () => {
+    const { user } = useAuth();
+    const router = useRouter();
+    const [baanName, setBaanName] = useState<string | null>(null);
+    useEffect(() => {
+        async function getBaanName(user: IUser | null): Promise<void> {
+            const baan = await getBaan(user?.baanId || '');
+            setBaanName(baan?.name || null);
+        }
+        getBaanName(user || null);
+    }, [baanName]);
+    return (
+        <div className="flex h-52 w-80 justify-center rounded-3xl bg-white ring-4 ring-white/40">
+            <div className="justify-cente mx-8 flex items-center">
+                <div className="relative aspect-square h-36 w-24 max-w-full rounded-xl ring-4 ring-blue-950">
+                    <Image
+                        src={user?.imageUrl || placeHolder}
```

Comment:
Files which store in `public` folder is already static file. so you can `/images/pfp-placeholder.svg` in src without import.